client/src/game/star.js

```diff
@@ -167,12 +173,11 @@ class Star extends EventEmitter {
         planetContainer.addChild(planetGraphics)
 
         planetContainer.pivot.set(distanceToStar, 0)
-        planetContainer.position.x = this.data.location.x
-        planetContainer.position.y = this.data.location.y
 
         let rotationSpeed = (planetCount - i) / rotationSpeedModifier
 
         this.app.ticker.add((delta) => {
+          //maybe check if visible? no need to rotate planets outside viewport
```

Comment:
Yes, but let's not do it as part of this. Stick a TODO on the comment so we can easily find it later.